arm-storage/2015-06-15/swagger/storage.json

```diff
@@ -283,23 +253,29 @@
         ],
         "responses": {
           "200": {
-            "description": "",
+            "description": "OK -- List of storage accounts was retrieved and returned successfully.",
```

Comment:
IMO: `OK -- List of storage accounts was retrieved and returned successfully` doesn't seem to best grammatically. 

How about `OK -- Storage accounts were retrieved successfully.`